### PR TITLE
feat: Updating the error messages on the login form

### PR DESCRIPTION
**Pull request by TomDowling**
## Changes
Changed the modal header text size for the auth modals as well as fixing the validation / error states for the inputs in login. As well as adding a toast when a user try's to register with an existing email
login-form

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1164 #done

### CODE REVIEW

File: packages/shared/src/components/auth/LoginForm.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ function LoginForm({
     const form = formToJson<LoginFormParams>(e.currentTarget);
     onPasswordLogin(form);
   };
+  const [hasFocused, setHasFocused] = useState(autoFocus);
```
Comment (rebelchris) [inline packages/shared/src/components/auth/LoginForm.tsx:46]: Technically the naming is in reverse here, might cause confusion down the line.
You set hasfocus to false, but actually you focussed it.

Maybe shouldFocus is better in that case?
Or negate the start to be reversed !autoFocus and set it to true

File: packages/shared/src/components/fields/PasswordField.tsx [hunk 2]
```diff
@@ -64,8 +64,8 @@ export function PasswordField({
       valueChanged={onChange}
       leftIcon={<LockIcon size={IconSize.Small} />}
       hint={!!value && showStrength ? hint : props.hint}
-      validityChanged={setIsValid}
-      valid={isValid || !props.hint}
+      validityChanged={() => !props.hint && setIsValid}
```
Comment (rebelchris) [inline packages/shared/src/components/fields/PasswordField.tsx:67]: It still looks like we miss the passing of the actual value to setIsValid as the () is empty.
Not sure it makes a difference for areas where we use an internal hint actually.
As for instance the hint space password field (unused atm)

**Review by rebelchris — CHANGES_REQUESTED** [review]
Neat, small comments but with those should be fine

File: packages/shared/src/components/auth/AuthDefault.tsx [hunk 3]
```diff
@@ -82,6 +83,8 @@ const AuthDefault = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [shouldLogin]);
 
+  const { displayToast } = useToastNotification();
```
Comment (rebelchris) [inline packages/shared/src/components/auth/AuthDefault.tsx:86]: Can you just move this to the top where all the other states are.
Should not be placed below render functions

**Review by rebelchris — CHANGES_REQUESTED** [review]
Small comment, otherwise we can go

**Review by rebelchris — APPROVED** [review]
Good from my side

**Review by sshanzel — APPROVED** [review]
Looks good to me. Just that one line I mentioned to be removed.

File: packages/shared/src/components/auth/LoginForm.tsx [hunk 4]
```diff
@@ -70,7 +72,6 @@ function LoginForm({
         value={email}
         data-testid="login_email"
         saveHintSpace
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/auth/LoginForm.tsx:74]: Should remain to keep the same space we use for all fields.
it was the easier fix to simply leave it although naming now is a bit funky.

**Pull request merged**